models/authorization.js

```diff
@@ -43,30 +43,17 @@ function can(user, feature, resource) {
   validateUser(user);
   validateFeature(feature);
 
-  let authorized = false;
+  if (!user?.features?.includes(feature)) return false;
```

Comment:
```suggestion
  if (!user.features.includes(feature)) return false;
```

Não precisa do encadeamento opcional, pois isso já é coberto pela função `validateUser`

Reply:
show, faz sentido! removido 🚀